projects/1.20/assets/dynamictrees/dynamictrees/lang/zh_cn.json

```
@@ -0,0 +1,143 @@
+{  
+	"itemGroup.dynamictrees": "有活力的树",
+
+  "entity.dynamictrees.falling_tree": "倒下的树",
+
+  "block.dynamictrees.oak_branch": "橡木",
+  "block.dynamictrees.azalea_branch": "杜鹃木",
+  "block.dynamictrees.spruce_branch": "云杉木",
+  "block.dynamictrees.birch_branch": "白桦木",
+  "block.dynamictrees.jungle_branch": "丛林木",
+  "block.dynamictrees.dark_oak_branch": "深色橡木",
+  "block.dynamictrees.acacia_branch": "金合欢木",
+  "block.dynamictrees.crimson_branch": "绯红菌木",
+  "block.dynamictrees.warped_branch": "诡异菌木",
+  "block.dynamictrees.mangrove_branch": "红木",
+  "block.dynamictrees.mangrove_roots": "红木",
+  "block.dynamictrees.cherry_branch": "樱花木",
+  "block.dynamictrees.rooty_water": "水根",
```

```suggestion
  "block.dynamictrees.oak_branch": "橡树",
  "block.dynamictrees.azalea_branch": "杜鹃树",
  "block.dynamictrees.spruce_branch": "云杉",
  "block.dynamictrees.birch_branch": "白桦",
  "block.dynamictrees.jungle_branch": "丛林树",
  "block.dynamictrees.dark_oak_branch": "深色橡树",
  "block.dynamictrees.acacia_branch": "金合欢树",
  "block.dynamictrees.crimson_branch": "巨型绯红菌",
  "block.dynamictrees.warped_branch": "巨型诡异菌",
  "block.dynamictrees.mangrove_branch": "红树",
  "block.dynamictrees.mangrove_roots": "红树",
  "block.dynamictrees.cherry_branch": "樱花树",
```
https://zh.minecraft.wiki/w/%E6%A0%91%E6%9C%A8
推荐按照地物名称来